pkg/configutils/static/99-embedded-cluster.conf

```diff
@@ -14,5 +14,5 @@ net.ipv4.conf.all.arp_ignore = 0
 # particularly when communicating with cluster services from the host. This is
 # because Kubernetes networking paths are not always seen as the "best reverse
 # path" by the kernel.
```

Comment:
I feel like this comment is now inaccurate, no?

Reply:
yes, let me update. ty